pkg/runtime/gomemlimit_linux.go

```diff
@@ -0,0 +1,52 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
```

Comment:
Do we need to have a build constraint to build this for Go 1.19 and older?

Reply:
Good idea yes, thanks! Added.